py/legacyanalysis/sersic-agn.py

```diff
@@ -0,0 +1,269 @@
+from tractor.galaxy import *
+from tractor import *
+from legacypipe.survey import *
+from legacypipe.coadds import quick_coadds
+from tractor.devagn import *
+from tractor.sersic import SersicGalaxy, SersicIndex
+from tractor.seragn import SersicAgnGalaxy
+from astrometry.util.file import pickle_to_file, unpickle_from_file
+
+import pylab as plt
+import numpy as np
+
+import matplotlib.gridspec as gridspec
+import os
+
+ima = dict(interpolation='nearest', origin='lower')
+
+class Duck(object):
+    pass
+
+def flux_string(br):
+    s = []
+    for band in 'grz':
+        flux = br.getFlux(band)
+        if flux <= 0:
+            s.append('%s=(%.2f nmy)' % (band, flux))
+        else:
+            s.append('%s=%.2f' % (band, NanoMaggies.nanomaggiesToMag(flux)))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['NanoMaggies' may be undefined, or defined from star imports: legacypipe.survey, tractor, tractor.devagn, tractor.galaxy](https://app.codacy.com/app/dstndstn/legacypipe/pullRequest?prid=4730106)